typhos/cache.py

```diff
@@ -3,7 +3,6 @@
 import fnmatch
 import functools
 import logging
-import ophyd
```

Comment:
it was me! :D my bad!

Reply:
Even worse, I pushed a commit that shouldn't be here:  52e2f7c ; re-force pushing to save face